observability/prometheusrules.jsonnet

```
@@ -264,105 +286,104 @@ local renderAlerts(name, environment, mixin) = {
-          metric: 'http_requests_total',
-          selectors: ['job="observatorium-observatorium-api"', 'handler="query_range"', 'code=~"^(2..|3..|5..)$"'],
-          errorSelectors: ['code=~"5.+"'],
-          target: 0.95,
-        }),
-      ],
-    },
-    {
-      name: 'rhobs-telemeter-api-metrics-read-latency.slo',
-      slos: [
-        slo.latencyburn({
-          alertName: 'TelemeterAPIMetricsReadLatencyErrorBudgetBurning',
-          alertMessage: 'API /query endpoint is burning too much error budget to gurantee latency SLOs',
-          metric: 'up_custom_query_duration_seconds_bucket',
-          selectors: ['query="query-path-sli-1M-samples"'],
-          latencyTarget: 2.0113571874999994,
-          latencyBudget: 0.9,
-        }),
-        slo.latencyburn({
-          alertName: 'TelemeterAPIMetricsReadLatencyErrorBudgetBurning',
-          alertMessage: 'API /query endpoint is burning too much error budget to gurantee latency SLOs',
-          metric: 'up_custom_query_duration_seconds_bucket',
-          selectors: ['query="query-path-sli-10M-samples"'],
-          latencyTarget: 10.761264004567169,
-          latencyBudget: 0.9,
-        }),
-        slo.latencyburn({
-          alertName: 'TelemeterAPIMetricsReadLatencyErrorBudgetBurning',
-          alertMessage: 'API /query endpoint is burning too much error budget to gurantee latency SLOs',
-          metric: 'up_custom_query_duration_seconds_bucket',
-          // We can't use !~ operator in these selectors
-          selectors: ['query="query-path-sli-100M-samples"'],
-          latencyTarget: 21.6447457021712,
-          latencyBudget: 0.9,
-        }),
-      ],
-    },
   ],
 
-  local telemeter = {
-    prometheusAlerts+:: {
-      groups: [
-        {
-          local slos = [
-            slo.alerts + slo.recordingrules
-            for slo in group.slos
-          ],
-
-          name: group.name,
-          rules: std.flattenArrays(slos),
-        }
-        for group in telemeterSLOs
-      ],
-    },
+  local apiSLOs = function(instance, upNamespace, apiJob) {
+    local apiJobSelector = 'job="' + apiJob + '"',
+    local upNamespaceSelector = 'namespace="' + upNamespace + '"',
+    slos: [
+      {
+        name: 'rhobs-' + instance + '-api-metrics-write-availability.slo',
+        slos: [
+          slo.errorburn({
+            alertName: 'APIMetricsWriteAvailabilityErrorBudgetBurning',
+            alertMessage: 'API /receive handler is burning too much error budget to gurantee availability SLOs',
+            metric: 'http_requests_total',
+            selectors: [apiJobSelector, 'handler=~"receive"', 'code=~"^(2..|3..|5..)$"'],
+            errorSelectors: ['code=~"5.+"'],
+            target: 0.95,
+          }),
+        ],
+      },
+      {
+        name: 'rhobs-' + instance + '-api-metrics-write-latency.slo',
+        slos: [
+          slo.latencyburn({
+            alertName: 'TelemeterAPIMetricsWriteLatencyErrorBudgetBurning',
```

I think here we want to drop the instance name too?
```suggestion
            alertName: 'APIMetricsWriteLatencyErrorBudgetBurning',
```

---

Yes! Good spot.